ext/fileinfo/tests/finfo_file_basic.phpt

```diff
@@ -24,5 +24,5 @@ try {
 *** Testing finfo_file() : basic functionality ***
 string(28) "text/x-php; charset=us-ascii"
 string(22) "PHP script, ASCII text"
-string(29) "text/x-file; charset=us-ascii"
+string(43) "magic text file for file(1) cmd, ASCII text"
```

Comment:
Why did this change (esp. in this commit?)

Reply:
I'm not really sure ngl. Will try to look into it a bit more

Reply:
Ah the reason was because I messed up the option restoring code.